### PR TITLE
Update of section loading a delimited text file

### LINKED ISSUE #4220: [delimiter text] Add Point{Z,M,ZM} geometry support (fixes #25645) (#31595)
Original commit: https://github.com/qgis/QGIS/commit/5df309447ebac64a8f58640472ffabf7d3b57179 by web-flow

[FEATURE][delimiter text] Add Point{Z,M,ZM} geometry support (fixes #25645)

### PR DESCRIPTION
**Pull request by lajunek**
Goal: Update the 3.10 documentation with the information about delimited text files containing also Z and M fields.

Ticket(s): fix #4220 

- [ ] Backport to LTR documentation is required

### CODE REVIEW

**Comment by lajunek** [comment]
@havatv sorry for the mistake, is it because of the screenshot ?

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 1]
```diff
@@ -401,10 +401,11 @@ Geometry definition
...
 * |radioButtonOn|:guilabel:`Point coordinates` and provide the :guilabel:`X
-  field` and :guilabel:`Y field` if the layer is of point geometry type and
-  contain such coordinate fields. If the coordinates are defined as
-  degrees/minutes/seconds, activate the |checkbox|:guilabel:`DMS coordinates`
-  checkbox;
+  field`, the :guilabel:`Y field`, :guilabel:`Z field` (for 3-dimensional data)
+  and the :guilabel:`M field`(for the measurement dimension) if the layer is of 
+  point geometry type and contains such coordinate fields. If the coordinates
+  are defined as degrees/minutes/seconds, activate the
+  |checkbox|:guilabel:`DMS coordinates`checkbox;
```
Comment (havatv) [inline source/docs/user_manual/managing_data_source/opening_data.rst]:
```suggestion
  |checkbox|:guilabel:`DMS coordinates` checkbox;
```

Comment (havatv) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: Could this be the cause of the travis problem?

**Comment by havatv** [comment]
> @havatv sorry for the mistake, is it because of the screenshot ?

I don't think so, but if file names are case sensitive, you will have to change the suffix to .png
(`delimited_text_dialog.PNG` -> `delimited_text_dialog.png`).

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 2]
```diff
@@ -401,10 +401,11 @@ Geometry definition
 Once the file is parsed, set :guilabel:`Geometry definition` to
 
 * |radioButtonOn|:guilabel:`Point coordinates` and provide the :guilabel:`X
-  field` and :guilabel:`Y field` if the layer is of point geometry type and
-  contain such coordinate fields. If the coordinates are defined as
-  degrees/minutes/seconds, activate the |checkbox|:guilabel:`DMS coordinates`
-  checkbox;
+  field`, the :guilabel:`Y field`, :guilabel:`Z field` (for 3-dimensional data)
+  and the :guilabel:`M field`(for the measurement dimension) if the layer is of
```
Comment (DelazJ) [inline source/docs/user_manual/managing_data_source/opening_data.rst]:
```suggestion
  and the :guilabel:`M field` (for the measurement dimension) if the layer is of 
```

File: source/docs/user_manual/managing_data_source/opening_data.rst (hunk 1 above)
Comment (DelazJ) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: Yes, that's (actually the one I highlighted, but this one would be next one if not fixed)

**Comment by DelazJ** [comment]
@lajunek follow the red cross and you'd find the reason of the failure.
About the screenshot, as said in the previous PR, this is case sensitive so using "PNG" while the existing file uses "png" won't make it replace the existing screenshot, it will just add a new but unused screenshot to the repo. You need to lower the case of the file extension.

**Comment by lajunek** [comment]
@DelazJ I was for a way to lower the file extension. Is it possible in GIMP to export a *.bmp file? I use this under win 10.

**Comment by lajunek** [comment]
@DelazJ o.k. think I found a solution

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 3]
```diff
@@ -336,10 +336,10 @@ Mesh layers (currently :file:`.grb`, :file:`.grb2`, :file:`.bin`, :file:`.grib`,
 Importing a delimited text file
 -------------------------------
 
-Delimited text file (e.g. :file:`.csv`, :file:`.txt`) can be loaded in QGIS
-using the tools described above. However, loaded this way, it'll show up like a
-simple table data. Sometimes, delimited text files can contain geometric data
-you'd want to visualize; this is what the |addDelimitedTextLayer|:guilabel:`Add
+Delimited text file (e.g. :file:`.txt`,:file:`.csv`, :file:`.dat`, :file: `.wkt`)
```
Comment (havatv) [inline source/docs/user_manual/managing_data_source/opening_data.rst]:
```suggestion
Delimited text files (e.g. :file:`.txt`, :file:`.csv`, :file:`.dat`, :file: `.wkt`) 
```

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 4]
```diff
@@ -336,10 +336,10 @@ Mesh layers (currently :file:`.grb`, :file:`.grb2`, :file:`.bin`, :file:`.grib`,
...
 -------------------------------
 
-Delimited text file (e.g. :file:`.csv`, :file:`.txt`) can be loaded in QGIS
-using the tools described above. However, loaded this way, it'll show up like a
-simple table data. Sometimes, delimited text files can contain geometric data
-you'd want to visualize; this is what the |addDelimitedTextLayer|:guilabel:`Add
+Delimited text file (e.g. :file:`.txt`,:file:`.csv`, :file:`.dat`, :file: `.wkt`) 
+can be loaded in QGIS using the tools described above. However, loaded this way,
+it'll show up like a simple table data. Sometimes, delimited text files can contain
+geometric data you'd want to visualize; this is what the |addDelimitedTextLayer|:guilabel:`Add
```
Comment (havatv) [inline source/docs/user_manual/managing_data_source/opening_data.rst]:
```suggestion
coordinates / geometries that you could want to visualize; this is what the
|addDelimitedTextLayer|:guilabel:`Add Delimited Text Layer` is designed for.
```

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 5]
```diff
@@ -336,10 +336,10 @@ Mesh layers (currently :file:`.grb`, :file:`.grb2`, :file:`.bin`, :file:`.grib`,
...
 Delimited Text Layer` is designed for.
```
Comment (havatv) [inline source/docs/user_manual/managing_data_source/opening_data.rst]:
```suggestion
```

**Comment by havatv** [comment]
The file "delimited_text_dialog.png" is 2.7 MB. While the one it replaces is only 94.8 KB. Could the size be reduced?

**Comment by havatv** [comment]
Thanks, @lajunek!

**Comment by havatv** [comment]
> The file "delimited_text_dialog.png" is 2.7 MB. While the one it replaces is only 94.8 KB. Could the size be reduced?

I have fixed the file size issue in #4659. Will merge that one as soon as Travis is happy.

**Pull request merged**